testResources/roots-option-project/__tests__/isAdult.spec.js

```diff
@@ -0,0 +1,7 @@
+const isAdult = require('../lib/isAdult.js');
```

Comment:
Cool to see an extra test that verifies the fix! Will it work with Jest 18?

Reply:
I believe so: https://travis-ci.org/stryker-mutator/stryker-jest-runner/jobs/296137148